### PR TITLE
Create and switch to vumi user

### PR DESCRIPTION
**Pull request by JayH5**
Now that praekeltfoundation/dockerfiles#33 is done

### CODE REVIEW

**Comment by JayH5** [comment]
Example `ps aux` output:
```
PID   USER     TIME   COMMAND
    1 root       0:00 dinit vumi-entrypoint.sh
    5 vumi       0:00 {twistd} /usr/local/bin/python2 /usr/local/bin/twistd --nodaemon --pidfile= vumi_worker --worker-class vumi.transports.telnet.TelnetServerTransport --set-option=telnet_port:9010 --set-option=transport_name:telnet
   24 root       0:00 ps aux
```

File: alpine/Dockerfile [hunk 1]
```diff
@@ -4,6 +4,9 @@ MAINTAINER Praekelt Foundation <dev@praekeltfoundation.org>
 COPY ./requirements.txt /requirements.txt
 RUN pip install -r /requirements.txt
 
+RUN addgroup vumi \
+    && adduser -S -G vumi vumi
```
Comment (jerith) [inline alpine/Dockerfile]: Does alpine's `adduser` not support the long args?

**Comment by jerith** [comment]
:+1:

File: alpine/Dockerfile (hunk 1 above)
Comment (JayH5) [inline alpine/Dockerfile]: BusyBox's `adduser` does not, unfortunately:
```
/ # addgroup vumi
/ # adduser --system --ingroup vumi vumi
adduser: unrecognized option: system
BusyBox v1.24.2 (2016-06-23 08:49:16 GMT) multi-call binary.

Usage: adduser [OPTIONS] USER [GROUP]

Create new user, or add USER to GROUP

	-h DIR		Home directory
	-g GECOS	GECOS field
	-s SHELL	Login shell
	-G GRP		Add user to existing group
	-S		Create a system user
	-D		Don't assign a password
	-H		Don't create home directory
	-u UID		User id
	-k SKEL		Skeleton directory (/etc/skel)
```

**Pull request merged**